src/components/input/Input.stories.tsx

```diff
@@ -38,4 +38,56 @@ storiesOf("Input", module)
                 onChange={action("changed")}
             />
         </div>
+    ))
+    .add("input box with label and helper text", () => (
+        <div>
+            <Input
+                name="somevalue"
```

Comment:
Will it be possible to send the position of the label as Left or Top?
And is it possible to specify variations such as icon left/right, password, IP address etc for input type?

Reply:
we do not have any properties to send position, we can send type, I have added a story for password.